topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio/tutorial.md

```diff
@@ -58,83 +58,85 @@ notebook:
 {% snippet topics/single-cell/faqs/notebook_warning.md %}
 
 # Setting your environment
-First thing's first, we need to load the packages we will be using. In order to use any functions of a package, we must first call the library of that package. In your console (likely in the lower left corner of your RStudio window), run the following lines of code to do so:
+First thing's first, we need to load the packages we will be using. In order to use any functions within a package, we must first call the library of that package. In your console (likely in the lower left corner of your RStudio window), run the following lines of code to do so:
 
 ```r
 library(Matrix)
 library(Seurat)
 library(dplyr)
 ```
 
-The packages are called--now let's get our data files moved from our Galaxy history and into our RStudio enviornment so that we can create a Seurat object.
+The packages are called--now let's get our data files moved from our Galaxy history and into the RStudio enviornment so that we can create a Seurat object.
```

Comment:
```suggestion
The packages are called--now let's get our data files moved from our Galaxy history and into the RStudio environment so that we can create a Seurat object.
```